training/cg_detr_dataset.py

```diff
@@ -188,9 +188,13 @@ def __getitem__(self, index):
             else:
                 if "relevant_windows" in meta: ## For Qvhighlights test set
                     model_inputs["span_labels"] = self.get_span_labels(meta["relevant_windows"], ctx_l)  # (#windows, 2)
-                    if self.dset_name == 'qvhighlight':
-                        model_inputs["saliency_pos_labels"], model_inputs["saliency_neg_labels"], model_inputs["saliency_all_labels"] = \
-                            self.get_saliency_labels_all(meta["relevant_clip_ids"], meta["saliency_scores"], ctx_l)
+                    if 'qvhighlight' in self.dset_name:
+                        if "subs_train" not in self.data_path:
```

Comment:
could you switch this if-else? I mean, instead of ""subs_train" **not** in self.data_path", please write like: "subs_train" in self.data_path.

Reply:
OK. I understood.